index.html

```diff
@@ -9,8 +8,15 @@
     <script src="js/jquery-scrolltofixed-min.js" type="text/javascript"></script>
     <link rel='stylesheet' href='js/jquery-ui-1.11.4.custom/jquery-ui.min.css' />
     <script type="text/javascript" src="js/stickynav.js"></script>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css">
+    <script src="//code.jquery.com/jquery-1.10.2.js"></script>
+    <script src="//code.jquery.com/ui/1.11.4/jquery-ui.js"></script>
+    <script src='js/jquery-1.11.3.min.js'></script>
+    <script src='js/jquery-ui-1.11.4.custom/external/jquery/jquery.js'></script>
+    <script src='js/jquery-ui-1.11.4.custom/jquery-ui.min.js'></script>
```

Comment:
some of those jquery references were for the droplist.. not exactly sure which ones. ill look into that.